enterprise_access/apps/api/v1/views/subsidy_access_policy.py

```diff
@@ -419,6 +465,11 @@ def redemption(self, request, *args, **kwargs):
             status=status.HTTP_200_OK,
         )
 
+    @extend_schema(
+        tags=['Subsidy Access Policy Redemption'],
+        summary='Can redeem.',
+        parameters=[serializers.SubsidyAccessPolicyCanRedeemRequestSerializer],
```

Comment:
I did a weird thing with this request serializer to inject the enterprise_customer_uuid URL location parameter into the serializer.  That probably makes the redoc result of this incorrectly represent enterprise_customer_uuid as a query parameter.  Probably what I should have done was inject it as a bit of serializer context or something other than a parameter...maybe you can do that here?

Reply:
oh wait, how the heck does it work (correctly appears as a path parameter): 
![image](https://user-images.githubusercontent.com/85151/235767746-413a7c42-26e5-433e-ba33-ad5f086241c3.png)

Reply:
no wait, enterprise_customer_uuid appears twice, the first one is probably inferred magically, and the second one borrowed from the serializer.